engine/engine.go

```diff
@@ -39,8 +39,8 @@ const (
 type Opts struct {
 	promql.EngineOpts
 
-	// DisableOptimizers disables Query optimizations using logicalPlan.DefaultOptimizers.
-	DisableOptimizers bool
+	// LogicalOptimizers are optimizers which are run in addition to the default ones.
```

Comment:
The comment looks wrong? 
```suggestion
	// LogicalOptimizers are optimizers that are run if the value is not nil. If it is nil then the default optimizers are run. Default optimizer list is available in the logicalplan package.
```